rust/astarte-message-hub-proto/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+# This file is part of Astarte.
+#
+# Copyright 2023 SECO Mind Srl
+#
+# SPDX-License-Identifier: CC0-1.0
+
+[package]
+name = "astarte-message-hub-proto"
+version = "0.5.1"
```

Comment:
Currently I kept  the same version of message-hub because before this repo they were the same project, so we can decide  if  follow the minor of the message hub, or mantain the version of this package independent, but the first version i think should be the same of message-hub.